massa-node/src/main.rs

```diff
@@ -372,13 +372,27 @@ async fn launch(
         max_denunciations_per_block_header: MAX_DENUNCIATIONS_PER_BLOCK_HEADER,
     };
 
+    // Creates an empty default store
+    let mip_stats_config = MipStatsConfig {
+        block_count_considered: MIP_STORE_STATS_BLOCK_CONSIDERED,
+        counters_max: MIP_STORE_STATS_COUNTERS_MAX,
+    };
+    let mut mip_store =
+        MipStore::try_from(([], mip_stats_config)).expect("Cannot create an empty MIP store");
```

Comment:
It should not be empty anymore as we have versioned address no?

Reply:
Was planning to update this when I move to high end testing, didn't get there yet.